### PR TITLE
fix(website): fix some layout issues on event pages

### PR DESCRIPTION
**Pull request by MrGreenTea**

### CODE REVIEW

**Comment by MrGreenTea** [comment]
Wanted to fix #189 but haven't found out how to fix it. Now this only fixes the unoptimized images

File: src/pages/events/[slug].astro [hunk 1]
```diff
@@ -165,12 +168,13 @@ const locationPictures = [
             <h2 class="font-serif text-2xl font-bold lg:text-4xl">Begleitung und Referierende</h2>
             <div class="-mt-4 grid grid-cols-1 gap-x-6 lg:-mt-6 lg:grid-cols-2 lg:gap-x-12">
               {event.team?.map(async (member) => (
-                <div class="row-span-3 mt-8 grid grid-rows-subgrid lg:mt-16">
-                  <img
-                    src={member.image.src.src}
-                    alt=""
-                    class="aspect-square rounded-lg object-cover"
-                    sizes="(max-width: 1024px) 100vw, 440px"
+                <div class="row-span-3 mt-8 lg:mt-16 lg:grid lg:grid-rows-subgrid">
```
Comment (Han2-Ro) [inline src/pages/events/[slug].astro:171]: This change seems to fix the spacing issue for mobile.

**Review by Han2-Ro — COMMENTED** [review]
Nice Image Optimization :)

File: src/pages/events/[slug].astro (hunk 1 above)
Comment (Han2-Ro) [reply src/pages/events/[slug].astro]: I just went one step further and removed `gird` and `gird-row-subgrid` in this line completely (not just for mobile) and it fixed the spacing. I don't really understand why this is making the difference.

Comment (MrGreenTea) [reply src/pages/events/[slug].astro]: Hm you're right, it looks way better. I believe I added it in the past to align the pictures, titles and bios with each other. But it seems it's not necessary and looks good without it

**Pull request merged**